samples/snippets/src/test/java/com/example/spanner/GetCommitStatsSampleIT.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.common.collect.ImmutableList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration tests for {@link GetCommitStatsSample}
+ */
+@RunWith(JUnit4.class)
+public class GetCommitStatsSampleIT {
+
+  private static String instanceId = System.getProperty("spanner.test.instance");
+  private static String databaseId = formatForTest(
+      System.getProperty("spanner.sample.database", "commitstatssample"));
```

Comment:
We generally tend to use environment variables over system properties for configuration in Java samples (See https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#project-configuration). This makes it easier to set these properties for CI tests.

Is there a specific reason we're using properties here?

Reply:
That is what we are doing for all other sample integration tests:

- [AsyncExamplesIT](https://github.com/googleapis/java-spanner/blob/master/samples/snippets/src/test/java/com/example/spanner/AsyncExamplesIT.java#L51-L53)
- [QuickstartSampleIT](https://github.com/googleapis/java-spanner/blob/master/samples/snippets/src/test/java/com/example/spanner/QuickstartSampleIT.java#L43-L44)
- [SpannerSampleIT](https://github.com/googleapis/java-spanner/blob/master/samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java#L51-L52)
- [SpannerStandaloneExamplesIT](https://github.com/googleapis/java-spanner/blob/master/samples/snippets/src/test/java/com/example/spanner/SpannerStandaloneExamplesIT.java#L50-L52)

We could change how we are doing this and move to environment variables, but I would prefer to do that in separate tech debt fixing PR. Would that be ok?

Reply:
Doing it in a separate PR should be fine. I'll approve this PR, but please open an issue to track the work for updating this and other samples.